File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.java
class Solution {
    public boolean check(int[] nums) {
        int countbreaks = 0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]>nums[(i+1)%nums.length]){
                countbreaks+=1;
            }
        }
        if(countbreaks>1){
            return false;
        }
        return true;
    }
}

File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        countbreak = 0
        for i in range(0,len(nums)):
            if nums[i]>nums[(i+1)%len(nums)]:
                countbreak+=1
        
        if countbreak >1 :
            return False
        
        return True